ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -512,6 +516,10 @@ void completeRead() {
         if (e == null) {
           updateChunkingAndSizeMetricsOnSuccessfulGet();
         } else {
+          logger.warn(
```

Comment:
Do we need to log `e` as well? The blob ID may also be useful to log in order to correlate with the public access log.